server/controllers/response.ts

```diff
@@ -155,3 +156,44 @@ export function getAllResponse(surveyId: string, done: (err: any, res?: Response
         }
     });
 }
+export async function downloadRaw (req: Request, res: Response, next: NextFunction) {
```

Comment:
```suggestion
export async function downloadRaw (req: Request, res: Response, _next: NextFunction): Promise<void> {
```

Async functions always return `Promise` types. Since we aren't returning anything from this function, the type parameter is `void`. That makes the return type of this function `Promise<void>`.